src/Controller/Controller.ts

```
@@ -736,8 +736,11 @@ class Controller {
 			case ShellJob.WAR:
 				propsGenerator = new WARStatusPropsGenerator(game as WARState);
 				break;
-			default:
+			case ShellJob.BLM:
 				propsGenerator = new BLMStatusPropsGenerator(game as BLMState);
+				break;
+			default:
+				propsGenerator = new StatusPropsGenerator(game);
```

should we raise an error here instead?

---

My undersanding is this (as well as a few other changes) makes sure the site doesn't crash when a job is only partially implemented

---

makes sense, a `console.warning` might be helpful though